chapter_common_operators/matmul.md

```diff
@@ -1,17 +1,17 @@
 # Matrix Multiplication
 
-Matrix Multiplication is one of the most widely operators in scientific computing. Let's implement its computation in this chapter.
+Matrix Multiplication is one of the most widely operators in scientific computing and deep learning,which is typically referred to as *GEMM* (GEneral Matrix Multiply). Let's implement its computation in this section.
```

Comment:
I'm fine to change chapter to section. If so, we need to update all md files.

Reply:
Yes, I am updating it accordingly along the way. I think using section is better as it is consistent with the name generated by `:numref`. Chapter can be used when we refer to the entire unit, i.e. every fold in the repo.